src/main/java/gmailgdogra/service/FormatXlsxService.java

```diff
@@ -0,0 +1,145 @@
+package gmailgdogra.service;
+
+import org.apache.poi.ss.usermodel.*;
```

Comment:
the good practice is to avoid wildcard imports so that you can always see that the import is from the correct package. To do this, enable single class import in Editor | Code Style | Java | Imports

Reply:
good, thanks